code/controllers/subsystem/ticker.dm

```diff
@@ -422,13 +422,12 @@ var/datum/subsystem/ticker/ticker
 	var/list/total_antagonists = list()
 	//Look into all mobs in world, dead or alive
 	for(var/datum/mind/Mind in minds)
```

Comment:
performance gained is performance gained, and it sticks in peoples heads the idea of istypeless-forloops, so in actually intensive situations they've already set up their loops correctly.